.github/CODEOWNERS

```diff
@@ -3,15 +3,17 @@
 /builtins-default-costs/ @anza-xyz/svm
 /compute-budget/ @anza-xyz/svm
 /compute-budget-instruction/ @anza-xyz/svm
-/fee/ @anza-xyz/svm
+/fee/ @anza-xyz/svm @anza-xyz/fees
```

Comment:
@pgarg66 - since both teams are owners, I think only one team needs to approve. Maybe we just sort of agree that we should both review now before giving actual approval.
Hopefully these files don't change much outside some of the refactorings I am planning to do immediately.

Reply:
That's an unfortunate implementation decision by Github